frontend_tests/puppeteer_tests/custom-profile.ts

```diff
@@ -9,18 +9,24 @@ const profile_field_row = "#admin_profile_fields_table tr:nth-last-child(2)";
 const profile_field_form = "#admin_profile_fields_table tr:nth-last-child(1)";
 
 async function test_add_new_profile_field(page: Page): Promise<void> {
+    await page.click("#add-custom-profile-field-btn");
+    await common.wait_for_micromodal_to_open(page);
+    assert.strictEqual(
+        await common.get_text_from_selector(page, ".dialog_heading"),
+        "Add a new custom profile field",
+    );
+    assert.strictEqual(
+        await common.get_text_from_selector(page, "#dialog_widget_modal .dialog_submit_button"),
+        "Save changes",
+    );
```

Comment:
I don't think testing this is necessary.

Reply:
We test other existing models like this, so I did the same for these new models too.